### ISSUE TITLE
Visual Studio Code throws language server error when opening a Cobol file.

### ISSUE THREAD

**Issue by edrubins** [openmainframeproject/cobol-programming-course#104]
I installed Visual Studio code on a new Windows 10 machine, for use with the Cobol course. Opening COBOL0001.cobol from the causes a `Starting client failed. Launching server using command /bin/java failed.` I them installed it in a Linux system, where it works.
	
	Environment:
	
	Windows 10 Home
	Visual Studio version 1.44.2
	IBM Z Open Editor version 0.4.9
	Zowe Explorer version 1.4.1
	Java: 
	java version "1.8.0_251"
	Java(TM) SE Runtime Environment (build 1.8.0_251-b08)
	Java HotSpot(TM) Client VM (build 25.251-b08, mixed mode, sharing)
        I've also tried a number of Java SDKs  including the IBM Java 8 from Eclipse and the AdoptOpen SDKs.

	
The log files contain the following entries, in order.
	
	1 Git Authentication Log - empty
	3 - Accounting Log - Empty
	
	2-Git Log
	Looking for git in: C:\Program Files\Git\cmd\git.exe
	Using git 2.25.1.windows.1 from C:\Program Files\Git\cmd\git.exe
	> git rev-parse --show-toplevel
	> git rev-parse --git-dir
	Open repository: c:\Users\edrub\Cobal\IBM\cobol-training-2
	> git status -z -u
	> git symbolic-ref --short HEAD
	> git rev-parse master
	> git rev-parse --symbolic-full-name master@{u}
	> git rev-list --left-right master...refs/remotes/origin/master
	> git for-each-ref --format %(refname) %(objectname) --sort -committerdate
	> git remote --verbose
	> git check-ignore -v -z --stdin
	Failed to watch ref 'c:\Users\edrub\Cobal\IBM\cobol-training-2\.git\refs\remotes\origin\master', is most likely packed.
	Error: ENOENT: no such file or directory, watch 'c:\Users\edrub\Cobal\IBM\cobol-training-2\.git\refs\remotes\origin\master'
		at FSWatcher.start (internal/fs/watchers.js:165:26)
		at Object.watch (fs.js:1270:11)
		at Object.t.watch (c:\Users\edrub\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\git\dist\main.js:1:598268)
		at T.updateTransientWatchers (c:\Users\edrub\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\git\dist\main.js:1:83870)
		at e.fire (c:\Users\edrub\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:48:517)
		at e.updateModelState (c:\Users\edrub\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\git\dist\main.js:1:102740)
	> git config --get commit.template
	> git show :Labs/cbl/CBL0002.cobol
	> git ls-files --stage -- C:\Users\edrub\Cobal\IBM\cobol-training-2\Labs\cbl\CBL0002.cobol
	> git cat-file -s cd90e16e6fe04ff4a9573f8a666fe395c324ef6c
	
	4 Cobal Language Server Log
	[Error - 8:27:42 PM] Starting client failed
	Launching server using command /bin/java failed.
	
Any help appreciated. The linux install has it's own problem..

**Comment by edrubins** [comment]
I completely removed VSCode from the Windows machine, including the AppData\Roaming\Code directory, the .zowe and .vscode code directories in my user directories. I then installed VSCode and added the Zowe extension.  I hovered my mouse over Zowe => Data Sets and clicked '+', adding a connection named CobolTester. I hovered my mouse over Zowe => Data Sets and clicked '+' a second time, expecting to see CobolTester in the dropdown below '+ Create a New Connection to z/OS. It wasn't there.

Following the Medium article by Jessielaine Punongbayan on accessing mainframes with Zowe
([https://medium.com/zowe/beginners-guide-how-to-access-mainframe-via-zowe-in-10-easy-steps-fbec14ed6ed2](url).) I clicked on the CobolTester Create New Data Set icon, selected Data Set Classic and entered DataSetOne for the data set name. The following errors resulted.

Error encountered when creating data set! z/OSMF REST API Error:
Rest API failure with HTTP(S) status 500
category: 8
rc:       860
reason:   2
message:  Dynamic allocation Error

 Error: z/OSMF REST API Error:
Rest API failure with HTTP(S) status 500
category: 8
rc:       860
reason:   2
message:  Dynamic allocation Error

Error running command zowe.createDataset: z/OSMF REST API Error:
Rest API failure with HTTP(S) status 500
category: 8
rc:       860
reason:   2
message:  Dynamic allocation Error

. This is likely caused by the extension that contributes zowe.createDataset.

The VSCode log files produced are attached.
[20200503T153925.zip](https://github.com/openmainframeproject/cobol-programming-course/files/4571479/20200503T153925.zip)
